src/mediaplayer.cs

```diff
@@ -2085,5 +2121,49 @@ interface MPMusicPlayerApplicationController
 		[Export ("performQueueTransaction:completionHandler:")]
 		void Perform (Action<MPMusicPlayerControllerMutableQueue> queueTransaction, Action<MPMusicPlayerControllerQueue, NSError> completionHandler);
 	}
+
+	[NoTV][NoMac]
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface MPMusicPlayerPlayParameters : NSSecureCoding {
+		[Export ("initWithDictionary:")]
+		IntPtr Constructor (NSDictionary<NSString, NSObject> dictionary);
+
+		[Export ("dictionary", ArgumentSemantic.Copy)]
+		NSDictionary<NSString, NSObject> Dictionary { get; }
```

Comment:
Same, if this dictionary can be strongified in the future using `NSDictionary` instead of the generic one will save us some pain